apis/management.cattle.io/v3/cluster_types.go

```diff
@@ -302,3 +303,8 @@ type IngressCapabilities struct {
 	IngressProvider      string `json:"ingressProvider,omitempty"`
 	CustomDefaultBackend bool   `json:"customDefaultBackend,omitempty"`
 }
+
+type RotateCertificateInput struct {
+	RotateCACertificates bool     `json:"rotateCACertificates,omitempty" norman:"default=false"`
```

Comment:
I don't think we need Rotate* prepending the fields; should be just CACertificates, and Components
default=false is not needed. All bools are false by default

Reply:
sure i will do this change